lib/statsd/instrument/client.rb

```
@@ -120,6 +120,8 @@ def increment(name, value = 1, sample_rate: nil, tags: nil, no_prefix: false)
   def measure(name, value = nil, sample_rate: nil, tags: nil, no_prefix: false, &block)
     if block_given?
       return latency(name, sample_rate: sample_rate, tags: tags, metric_type: :ms, no_prefix: no_prefix, &block)
+    elsif value.nil?
```

Should we raise an error to avoid confusion if a block and a value is given ?